x/house/types/ticket.go

```diff
@@ -0,0 +1,38 @@
+package types
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// Validate validates market add ticket payload.
```

Comment:
comments mention market where as it should be house

Reply:
done